src/components/pokemons.js

```diff
@@ -1,20 +1,25 @@
 import './pokemons.scss';
-import { createElement, appendContent } from '../lib/dom';
+import { createElement } from '../lib/dom';
 
 export function createPokemons(props) {
   const container = createElement('div', {
     className: 'pokemons'
   });
 
-  props.items.forEach(item => {
+  props.items.forEach((item, pokemonId) => {
     const element = createElement('div', {
       innerText: item,
       className: 'pokemon'
     });
+    const pokemonImage = createElement('img', {
+      className: 'pokemonImage',
+      src: `https://pokeres.bastionbot.org/images/pokemon/${pokemonId + 1}.png`
+    });
```

Comment:
![image](https://user-images.githubusercontent.com/54837109/76397779-faea2e80-637b-11ea-9c39-792f037e00af.png)

Just added the index as pokemonId to change the number of each pokemon